compat-data-schema.md

```diff
@@ -0,0 +1,365 @@
+      },
+      …
+    }
+
+ That way, each feature is uniquely identifier, independently of the file it is
+ defined in.
+
+ The hierarchy of identifiers is not defined inside the schema. It is a
+ convention of the project using the schema.
+
+ In the MDN browser-compat case, it is:
+
+ _Note: this list will evolve as we migrate our data_
+
+     "data": {
+       "css": {
+         "properties": {…}
+         "pseudo-classes": {…}
+         "pseudo-elements": {…}
+         "at-rules": {…}
+       },
+       "html": {
+         "elements": {
+           …,
+           "<element-name>": {
+             …,
+             "<attribute-name>": {…},
+             …
+           }
+         }
+         "global_attributes": {…}  
+       }
+     }
+
+### Feature description
+
+A feature is represented by an identifier containing the `"__compat"` property.
+In this compat property, you'll find the list of sub-features associated to the
+feature.
+
+A feature has at least one sub-feature, representing the basic support. It is
+always named `"basic_support"`.
+
+The basic support feature represents the minimal set of functionality included
+when a feature is qualified of 'supported'. What this represents depends of the
+evolution of the feature over time, both in term of specification and of browser
+support. Another way of seeing it is to consider `"basic_support"` as representing all
+the functionality of the feature that doesn't have its own sub-feature(s).
+
+### Sub-feature
+
+A sub-feature is the basic entity having browser compatibility information. As
+explained in the previous paragraph, any feature has at least one sub-feature
+called `'basic_support'`, but it may many more.
+
+A sub-feature may have three properties.
+
+* __Sub-feature description__ contained in the `"desc"` property. It is a
+`string` that contains a human-readable description of the sub-feature. As it is
+intended to be used as a kind of caption or title for the feature, keep it short.
+the `<code>` and `<a>`, as well as the macros `{{cssxref}}`, `{{HTMLElement}}`,
+`{{htmlattrxref}}`, and `{{domxref}}` can be used. See the localization section
+below for an explanation about how this string will be localized.
+* __Compat information__ contained in the `"support"` property. It contains an
+object listing the compat information for each browser. (See the description
+below.)
+* __Status information__ contained in the `"status"` object. It contains the
+information about the stability of the sub-feature: Is it a functionality that
+is standard? Is it stable? Has it been deprecated and shouldn't be used anymore
+(See the description below.)
+
+### The `"support"` object
+Each sub-feature has support information. For each browser identifier, it
+contains a compat object with the information about versions, prefixes or
+alternate names, as well as notes.
+
+The currently accepted browser identifiers are:
+* `"webview_android"`, Webview, the former stock browser on Android,
```

Comment:
Hey @jpmedley. I suggested this, too: https://github.com/mdn/browser-compat-data/pull/86#pullrequestreview-35465721, and Florian just filed an issue for it: https://github.com/mdn/browser-compat-data/issues/168. I'm sure it would be great to get your input into what the format should be for Chrome.